ansys/mapdl/core/launcher.py

```diff
@@ -901,7 +900,6 @@ def launch_mapdl(exec_file=None, run_location=None, jobname='file',
         start_parm['start_timeout'] = start_timeout
     else:
         start_parm['ram'] = ram
-        start_parm['custom_bin'] = custom_bin
```

Comment:
This line has been re-added somewhere and caused issue #493

Reply:
Resolved in latest release.